code/modules/clothing/suits/ego_gear/non_abnormality/rcorp.dm

```diff
@@ -108,12 +107,36 @@
 	icon_state = "roadrunner_captain"
 	slowdown = -0.25
 
-
 /obj/item/clothing/suit/armor/ego_gear/rabbit/rat
 	name = "\improper rat suit"
 	desc = "An armored combat suit worn by R-Corporation skirmishers."
 	icon_state = "rat"
 
-
-
-
+//Rabbit specific NVs that break on removal
+/obj/item/clothing/glasses/night/rabbit
+	desc = "These goggles let you see in the dark perfectly. Sadly they're quite delicate, trying to remove them now will break them for sure."
+
+/obj/item/clothing/glasses/night/rabbit/Destroy()
+	dropped()
```

Comment:
Why does it call dropped() here?

Reply:
This is how its done in the ego_hat item so I assumed this is the way we figured out how to do it.